Package.swift

```diff
@@ -40,7 +40,7 @@ let package = Package(
     ),
     .binaryTarget(name: "TyroTapToPaySDK",
                   url: "https://tap-to-pay.connect.tyro.com/tyro/0.12.0/TyroTapToPaySDK.xcframework.zip",
-                  checksum: "282a22d834215a2a42b16dce36817409a40b2f17a36cb0867215c3570556a506"),
+                  checksum: "ce4a9c3d006f17fcee16cb13bfb4edaa3b912fe8b948aec22bc5d982ae1332c3"),
```

Comment:
we should only update these values and bump the version number after releasing the sdk

Reply:
Good point. I did not know that process